### PR TITLE
[FR] Add `Countries_that_participated_in_osu!_tournaments`

### PR DESCRIPTION
**Pull request by Biokax**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+
+### ![][osu!] classement osu!
+
+| Rang | Pays | ![][GCrown] | ![][SCrown] | ![][BCrown] |
+| :-: | :-- | :-: | :-: | :-: |
+| 1 | ![][flag_US] États-Unis | 6 | 1 |  |
+| 2 | ![][flag_KR] Corée du Sud | 3 | 1 | 3 |
+| 3 | ![][flag_JP] Japon | 1 | 2 |  |
+| 4 | ![][flag_PL] Pologne | 1 | 1 | 2 |
+| 5 | ![][flag_TW] Taiwan | 1 | 1 | 1 |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md:43]:
```suggestion
| 5 | ![][flag_TW] Taïwan | 1 | 1 | 1 |
```

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+| ![][flag_RO] Roumanie | ![][false] | ![][false] | ![][false] | ![][false] | ![][true] | ![][false] | ![][true] | ![][false] | ![][false] |
+| ![][flag_RU] Fédération de Russie | ![][false] | ![][false] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] |
+| ![][flag_SA] Arabie Saoudite | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][true] | ![][false] |
+| ![][flag_SG] Singapour | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][false] | ![][true] | ![][true] |
+| ![][flag_KR] Corée du Sud | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] |
+| ![][flag_ES] Espagne | ![][true] | ![][true] | ![][true] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] | ![][false] |
+| ![][flag_SE] Suède | ![][false] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] |
+| ![][flag_TW] Taiwan | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] |
```
Comment (Ganondorf-1) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md:252]:
```suggestion
| ![][flag_TW] Taïwan | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] | ![][true] |
```

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+[flag_PT]: /wiki/shared/flag/PT.gif "Portugal"
+[flag_RO]: /wiki/shared/flag/RO.gif "Roumanie"
+[flag_RU]: /wiki/shared/flag/RU.gif "Fédération de Russie"
+[flag_SA]: /wiki/shared/flag/SA.gif "Arabie Saoudite"
+[flag_SE]: /wiki/shared/flag/SE.gif "Suède"
+[flag_SG]: /wiki/shared/flag/SG.gif "Singapour"
+[flag_SK]: /wiki/shared/flag/SK.gif "Slovaquie"
+[flag_TH]: /wiki/shared/flag/TH.gif "Thaïlande"
+[flag_TR]: /wiki/shared/flag/TR.gif "Turquie"
+[flag_TW]: /wiki/shared/flag/TW.gif "Taiwan"
```
Comment (Ganondorf-1) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md:382]:
```suggestion
[flag_TW]: /wiki/shared/flag/TW.gif "Taïwan"
```

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+| 5 | ![][flag_JP] Japon |  | 2 |  |
+| 5 | ![][flag_PH] Philippines |  | 2 |  |
+| 6 | ![][flag_MY] Malaisie |  | 1 | 5 |
+| 7 | ![][flag_TH] Thaïlande |  | 1 |  |
+| 8 | ![][flag_ID] Indonésie |  |  | 1 |
+| 8 | ![][flag_GB] Royaume-Uni |  |  | 1 |
+
+## Participants
+
+### ![][osu!] osu! World Cup
```
Comment (Ganondorf-1) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md:97]: À partir d'ici, il serait pratique d'avoir les listes des pays triées par ordre alphabétique

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+| 12 | ![][flag_MY] Malaisie |  | 1 | 5 |
+| 13 | ![][flag_GB] Royaume-Uni |  | 1 | 4 |
+| 14 | ![][flag_ID] Indonésie |  | 1 | 3 |
+| 15 | ![][flag_NL] Pays-Bas |  | 1 |  |
+| 15 | ![][flag_TH] Thaïlande |  | 1 |  |
+| 16 | ![][flag_CA] Canada |  |  | 3 |
+| 17 | ![][flag_FR] France |  |  | 2 |
+| 18 | ![][flag_SE] Suède |  |  | 1 |
+
+### ![][osu!] classement osu!
```
Comment (DoryanR) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md:35]:
```suggestion
### ![][osu!] Classement osu!
```

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md [hunk 6]
```diff
@@ -0,0 +1,387 @@
...
+| 4 | ![][flag_PL] Pologne | 1 | 1 | 2 |
+| 5 | ![][flag_TW] Taiwan | 1 | 1 | 1 |
+| 6 | ![][flag_DE] Allemagne |  | 2 | 2 |
+| 7 | ![][flag_CN] Chine |  | 2 |  |
+| 8 | ![][flag_GB] Royaume-Uni |  | 1 | 2 |
+| 9 | ![][flag_NL] Pays-Bas |  | 1 |  |
+| 10 | ![][flag_BR] Brésil |  |  | 1 |
+| 10 | ![][flag_CA] Canada |  |  | 1 |
+
+### ![][osu!taiko] classement osu!taiko
```
Comment (DoryanR) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md:51]:
```suggestion
### ![][osu!taiko] Classement osu!taiko
```

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md [hunk 7]
```diff
@@ -0,0 +1,387 @@
...
+| 3 | ![][flag_HK] Hong Kong |  | 3 | 1 |
+| 3 | ![][flag_KR] Corée du Sud |  | 3 | 1 |
+| 4 | ![][flag_CA] Canada |  |  | 2 |
+| 4 | ![][flag_FR] France |  |  | 2 |
+| 5 | ![][flag_BR] Brésil |  |  | 1 |
+| 5 | ![][flag_CN] Chine |  |  | 1 |
+| 5 | ![][flag_ID] Indonésie |  |  | 1 |
+| 5 | ![][flag_GB] Royaume-Uni |  |  | 1 |
+
+### ![][osu!catch] classement osu!catch
```
Comment (DoryanR) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md:66]:
```suggestion
### ![][osu!catch] Classement osu!catch
```

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md [hunk 8]
```diff
@@ -0,0 +1,387 @@
+# Pays ayant participé aux tournois d'osu!
+
+Cet article établit une liste de pays ayant déjà participé dans des tournois officiels [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), et [osu!mania](/wiki/Game_mode/osu!mania). Il fournit également le classement des équipes ayant atteint le top 3.
```
Comment (DoryanR) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/fr.md:3]:
```suggestion
Cet article établit une liste de pays ayant déjà participé dans des tournois officiels des modes [osu!](/wiki/Game_mode/osu!), [osu!taiko](/wiki/Game_mode/osu!taiko), [osu!catch](/wiki/Game_mode/osu!catch), et [osu!mania](/wiki/Game_mode/osu!mania). Il fournit également le classement des équipes ayant atteint le top 3.
```

**Review by DoryanR — CHANGES_REQUESTED** [review]

**Review by DoryanR — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**